src/index.html

```diff
@@ -2526,6 +2533,58 @@ <h3>Examples:</h3>
 
 <!-- END TOKENS =============================================== END TOKENS -->
 
+<!-- ACTIVITY LOG =============================================== ACTIVITY LOG -->
+  <style>
+    #dialog_activity_log h2 {
+      font-size: 40px;
+      margin: 8px;
+      text-align: center;
+    }
+    #activity_log li {
+      background: #ddd;
+      border-radius: 5px;
+      display: flex;
+      font-size: 40px;
+      height: 80px;
+      justify-content: space-between;
+      list-style: none;
+      margin-bottom: 4px;
+      padding: 9px 9px 0 9px;
+      text-align: right;
+    }
+    #activity_log li:nth-child(even) {
+      background: #bbb;
+    }
+    #activity_log .activity_log_timestamp {
+      display: flex;
+      align-items: center;
+      font-size: 24px;
+    }
+    #activity_log svg {
+      vertical-align: bottom;
+    }
+  </style>
+
+  <template id="template_activity_item">
+    <li>
+      <div class="activity_item_timestamp"></div>
+      <div>
+        <span id="activity_item_player_name"></span>
+        <span id="activity_item_player_action"></span>
```

Comment:
Because these get cloned over and over, using `class` is going to be better semantics here, rather than `id`.  And the selectors in the javascript will need to change from `#` to `.`

Reply:
Yeah, I thought class would be better here, too. I found a StackOverflow that led me to believe that id would be a fine selector here, but in rereading it, I think I misread it. I'll make the change. 
https://stackoverflow.com/questions/9454645/does-id-have-to-be-unique-in-the-whole-page#:~:text=No.,6%2C%202019%2C%20clearly%20states%3A&text=When%20specified%20on%20HTML%20elements,may%20have%20several%20DOM%20trees.